proto/v1/event.proto

```diff
@@ -0,0 +1,62 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+option go_package = "./proto";
+package datacommons.v1;
+
+message EventCollection {
+  message GeoLocation {
+    message Point {
+      optional double latitude = 1;
+      optional double longitude = 2;
+    }
+    message Polygon { optional string geo_json = 1; }
+    oneof geo {
+      Point point = 1;
+      Polygon polygon = 2;
+    }
+  }
```

Comment:
I wonder should this proto be outside "EventCollection" since it feels like a fairly standalone concept?

Reply:
Maybe make PropVal and Event also top level proto, so they can potentially be reused for more APIs

Reply:
This is to exactly match the the protos in .../proto/prophet/cache_data.proto, so that we can directly pass the value in cache to API client... We could try to make a change in the cache first before modifying here.